api/controllers/EventController.js

```diff
@@ -35,7 +35,31 @@ var EventController = {
     
     delete: function (req, res) {
         
+    },
+
+    //GET /event/{id} operationId
+    getOne: function(req, res) {
+        var id = req.swagger.params.id.value;
+        Event.findById(id, function (e, event) {
+            if (e) return res.status(500).json(e.message);
+
+            res.json({eventDetail: event});
+            })     
+    },
+
+    //GET /event/{page},{pageSize} operationId
+    getOnePage: function(req, res) {
```

Comment:
can you rename to `getList` or `getEvents`so when you search for "getOne" only the the detail one shows up